src/sentry/static/sentry/app/components/stackedBarChart.jsx

```diff
@@ -169,7 +163,7 @@ class StackedBarChart extends React.Component {
   timeLabelAsRange(interval, point) {
     const timeMoment = moment(point.x * 1000);
     const nextMoment = timeMoment.clone().add(interval - 1, 'second');
-    const format = this.use24Hours() ? 'MMM Do, HH:mm' : 'MMM Do, h:mm a';
+    const format = use24Hours() ? 'MMM Do, HH:mm' : 'MMM Do, h:mm a';
```

Comment:
is `LT` == `h:mm a`? (I can't directly link, but: https://momentjs.com/)

Reply:
LT provides explicit locale support, which I'm assuming 'h:mm a' doesn't: https://momentjscom.readthedocs.io/en/latest/moment/06-i18n/01-changing-locale/

Locale support seems like a strict improvement in functionality, so I don't see any reason to not change this to 'LT'...